src/products/firefox_desktop/fluent_migrations.md

```diff
@@ -56,30 +65,35 @@ The last step here is to make sure that the current sync process has completed,
 
 It’s important to make sure that there is no sync in progress when upgrading the worker, because that will kill any pending process.
 
-### Push updates to gecko-strings
-
-Assuming the new strings are available in the [quarantine repository](https://hg.mozilla.org/users/axel_mozilla.com/gecko-strings-quarantine), it’s now time to push the updates to the official [gecko-strings repository](https://hg.mozilla.org/l10n/gecko-strings).
-
-Pontoon relies on a [different repository](https://hg.mozilla.org/users/m_owca.info/firefox-central/), that is generated every 20 minutes based on the content of `gecko-strings`. That’s why it’s useful to run this update before starting the actual migrations.
-
-### Run migrations
+### Test the migrations locally
 
 A series of tools and helpers to run migrations is available in [this repository](https://github.com/flodolo/fluent-migrations):
 * Clone the repository locally and follow the instructions available in the [README](https://github.com/flodolo/fluent-migrations/blob/master/README.md) to set the configuration file.
-* Add the new migration recipe in the `recipes` folder. The script will look for any Python file starting with `bug_` in that folder. More information about the folder’s structure are available in the repository’s README.
+* Add the new migration recipes in the `recipes` folder. The script will look for any Python file starting with `bug_` in that folder, so it’s possible to run multiple migrations in one pass. More information about the folder’s structure are available in the repository’s README.
 
-At this point, while the migration has been tested as part of the review before landing, it’s always good to run the migration against one locale, e.g. `it`:
+At this point, while each migration has been tested as part of the review before landing, it’s always good to run the migration against one locale, e.g. `it`:
 * Update the locale’s repository, check the results of `compare-locales`. Assuming l10n repositories are cloned in `~/migrations/locales`, and the quarantine repository in `~/migrations/gecko-strings-quarantine`, the command to run is `compare-locales ~/migrations/gecko-strings-quarantine/_configs/browser.toml ~/migrations/locales it`. Save the output in order to compare it with the results after the migration.
 * Run the migration, without pushing, only for `it`: `./scripts/migrate it wet-run`. Then run compare-locales again, and check if the results are as expected: migrated strings should not appear as missing anymore, and there should be no errors.
-* If everything looks good, re-run the script adding the push option `./scripts/migrate it wet-run push`.
+* If everything looks as expected, re-run the script adding the push option `./scripts/migrate it wet-run push`.
```

Comment:
I'd leave the `wet-run push` to the all-locales run below, so that it stays in the flow of not touching upstream unless quarantine is OK.

Reply:
I think it should be kept:
- Quarantine is already checked at this point, and Pontoon's sync is disabled.
- You've already committed the changes to the repository in the previous step, because you need them to run `compare-locales`. Here you're just pushing them out without going into the repo and running `hg push`.
- By removing this step, you risk conflicts unnecessarily. By the time the script has reached Italian again, you might have conflicts.